MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -21,7 +21,7 @@ public class CarPlayNavigationViewController: UIViewController {
     
     @objc public var drivingSide: DrivingSide = .right
     
-    var navService: NavigationService
+    @objc public var navigationService: NavigationService
```

Comment:
Add some documentation here and add a blurb in the changelog about this and the new delegate method.